check_for_new_releases.py

```diff
@@ -0,0 +1,68 @@
+import os
+import re
+import yaml
+from github import Github
+from urllib.parse import urljoin
+
+
+def get_subdirectories(d):
+    # https://stackoverflow.com/a/973492
+    return [o for o in os.listdir(d) 
+                    if os.path.isdir(os.path.join(d,o))]
+
+if __name__ == '__main__':
+    directories = get_subdirectories('./')
+    directories = [d for d in directories if not d.startswith('.') and not d == 'workflow']
+    
+    g = Github()
+    
+    for d in directories:
+        latest_version = get_subdirectories(d)[-1]
+        with open(os.path.join(d, latest_version, 'artifacthub-pkg.yml'), 'r') as stream:
+            try:
+                artifacthub_config = yaml.safe_load(stream)
+                repository_name = [link['url'] for link in artifacthub_config['links'] if link['name'] == 'Source'][0].replace('https://github.com/', '').partition('/tree')[0]
```

Comment:
Please refactor this line into a function with multiple lines of code and add a comment.
This improves readability and maintainability.

e.g.:
```python
# Get Repo URL from artifacthub config (Source URL)
def get_repo_url(artifacthub_config):
    for link in artifacthub_config['links']:
        if link['name'] == 'Source':
            return link['url']
```